config/sonic-nas.sources.list

```diff
@@ -1,2 +1,3 @@
 deb https://dell-networking.bintray.com/sonic-apt/ jessie main
+deb http://10.14.1.29/sonic unstable main
```

Comment:
hi cliff, there are many conflict in the pull request, can you update the scripts based the current master and update this commit?